ocs_ci/helpers/tlsprofile_helper.py

```
@@ -1106,3 +1244,552 @@ def _cleared():
+        for pod in pods:
+            container_statuses = pod.get("status", {}).get("containerStatuses") or []
+            if not container_statuses:
+                return False
+            if not all(cs.get("ready") for cs in container_statuses):
+                return False
+        return True
+
+    TimeoutSampler(timeout, sleep, _ready).wait_for_func_value(True)
+
+
+def assert_ocs_client_operator_https_tls_applied(
```

Ruff (0.16.2)

[warning] Too many branches (14 > 12)

(PLR0912)